releasenotes/notes/use-tls-1.2-by-default-e2983c21056c3359.yaml

```diff
@@ -0,0 +1,10 @@
+---
+enhancements:
```

Comment:
I think this is more an `upgrade` than `enhancements`, WDYT ?

Reply:
Yeah, I agree ! Thanks 🙏

Reply:
Hmm - I think this would be an enhancement - we're not really upgrading a piece of the agent but just bumping a setting

Reply:
From Reno:

```
Upgrade notes should be rare: only list known/potential breaking changes,
or major behaviorial changes that require user action before the upgrade.
Notes here must include steps that users can follow to 1. know if they're
affected and 2. handle the change gracefully on their end.
```

It's not a major / critical change but actually it fits with the fact that it might requires some users to change their configuration to keep their setup working (so potential breaking changes). Have no strong opinion on this one as the number of customers affected should be extra low.